# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/android/GfxInfo.cpp
# Commit: dd00e2da3a0f
# Full Hash: dd00e2da3a0f11a7cc0af9e5ece21a339f4393eb
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 69cc1de6f754 -r dd00e2da3a0f widget/android/GfxInfo.cpp
--- a/widget/android/GfxInfo.cpp	Wed Jun 10 21:57:19 2020 +0000
+++ b/widget/android/GfxInfo.cpp	Wed Jun 10 22:21:02 2020 +0000
@@ -65,7 +65,7 @@
     RefPtr<gl::GLContext> gl;
     nsCString discardFailureId;
     gl = gl::GLContextProvider::CreateHeadless(
-        gl::CreateContextFlags::REQUIRE_COMPAT_PROFILE, &discardFailureId);
+        {gl::CreateContextFlags::REQUIRE_COMPAT_PROFILE}, &discardFailureId);
 
     if (!gl) {
       // Setting mReady to true here means that we won't retry. Everything will
